### DISCUSSION TITLE
Errors are swallowed with Fastify

### DISCUSSION THREAD

**Discussion by vjau** [trpc/trpc#6302] [Q&A]
With the fastify adapters, errors are swallowed on the server (no trace of errors anywhere in the logs), but ends up appearing on the client in full disclosure, even through database errors, even in production. 
I would prefer the opposite ;)
What am i doing wrong ?

**Comment by KATT** [comment]
Fastify is one of those adapters that were added by external contributors but no one in the core team is using it

Sounds like a bug, please open an issue

**Comment by vjau** [comment]
Ok, i was doing it wrong.
1) There is an option when registering the fastify adapter plugin.
If you don't set the onError handler, the errors will be swallowed server side. For some reason, this handler was not set in my projects although it's written in the documentation. Either the docs have changed since i started using trpc, or this option didn't exist before, but when setting it, errors are not swallowed.
2) When trying to do a repro with server in production mode, i was not able to reproduce the full stacks that shows in the client.
Sorry for the confusion.
